cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,74 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="function")
```

Comment:
Any particular reason why this fixture is marked function scoped and not module scoped (considering that all the tests use this fixture) ?

Reply:
There is no particular reason, I required the config manager to be created for every test, so I was using function scope, but I guess using module scope doesn't make much difference either. Will change the scope.